.changeset/chilly-humans-report.md

```diff
@@ -0,0 +1,5 @@
+---
+'@guardian/source-react-components': minor
+---
+
+Adds an `isLoading` prop to the Button component. This replaces any icon rendered on the button with a spinner (`SvgSpinner`). The spinner will have colours that match the background and text colours of the button. The Button `disabled` state is also extended to show a `not-allowed` cursor and set the opacity to 0.4 so that the disabled state is indicated visually"
```

Comment:
```suggestion
Adds an `isLoading` prop to the Button component. This replaces any icon rendered on the button with a spinner (`SvgSpinner`). The spinner will have colours that match the background and text colours of the button. The Button `disabled` state is also extended to show a `not-allowed` cursor so that the disabled state is indicated visually"
```